frontends/election-manager/src/app.test.tsx

```diff
@@ -585,19 +587,18 @@ test('tabulating CVRs', async () => {
   const modal = await screen.findByRole('alertdialog');
   fireEvent.click(within(modal).getByText('Mark Tally Results as Official'));
 
-  // Report title should be rendered 3 times - app, preview, and printed
+  // Report title should be rendered 2 times - app and preview
   await waitFor(() => {
     expect(
       getAllByText('Official Mock General Election Choctaw 2020 Tally Report')
         .length
-    ).toBe(3);
+    ).toBe(2);
   });
 
-  // TODO: Snapshots without clear definition of what they are for cause future developers to have to figure out what the test is for each time this test breaks.
-  const printableArea1 = getByTestId('printable-area');
-  expect(
-    within(printableArea1).getByTestId('election-full-tally-report')
-  ).toMatchSnapshot();
+  userEvent.click(screen.getByText('Print Report'));
+  await screen.findByText('Printing');
+  // Snapshot printed report to detect changes in results or formatting
+  await expectPrintToMatchSnapshot();
```

Comment:
Taking out this `TODO` - I think printed elements are one area where snapshots make sense. Now we have a better API to explicitly snapshot the print.